install_config/oab_broker_configuration.adoc

```diff
@@ -0,0 +1,604 @@
+[[install-config-oab-broker-config]]
+= OpenShift Ansible Broker Configuration
+{product-author}
+{product-version}
+:icons: font
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+== Overview
+
+When the
+xref:../architecture/service_catalog/ansible_service_broker.adoc#arch-ansible-service-broker[OpenShift
+Ansible broker] (OAB) is deployed in a cluster, its behavior is largely dictated
+by the broker's configuration file loaded on startup. The broker's configuration
+is stored as a ConfigMap object in the broker's namespace
+(*openshift-ansible-service-broker* by default).
```

Comment:
Updated the default namespace here to match the `-n` options in the below commands, per QE feedback. cc @dymurray